src/Configuration/Config.NewtonsoftJson/ref/Microsoft.Extensions.Configuration.NewtonsoftJson.netstandard2.0.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Configuration
+{
+    public static partial class NewtonsoftJsonConfigurationExtensions
+    {
+        public static Microsoft.Extensions.Configuration.IConfigurationBuilder AddNewtonsoftJsonFile(this Microsoft.Extensions.Configuration.IConfigurationBuilder builder, Microsoft.Extensions.FileProviders.IFileProvider provider, string path, bool optional, bool reloadOnChange) { throw null; }
```

Comment:
Is "NewtonsoftJson" how we are going to refer to this everywhere we need it? (i.e. in other packages.)

Reply:
I just went with what MVC did: https://github.com/aspnet/AspNetCore/tree/master/src/Mvc/Mvc.NewtonsoftJson

Reply:
Cool. Just wanted us to be consistent.